frontend/src/v5/ui/components/dashboard/dashboardList/dashboardListHeaderLabel/dashboardListHeaderLabel.component.tsx

```diff
@@ -26,19 +26,31 @@ type IDashboardListHeaderLabel = {
 	sort?: boolean;
 	onClick?: Dispatch<void>;
 	width?: number;
+	tabletWidth?: number;
```

Comment:
This defines the width of the component when it's being displayed in a tablet, right? If that's the case can we rename this property to widthOnTablet? 
Or the other option is making width like this

```suggestion
	width?: number |  DisplaysWidth;
```

where DisplayWitdth is something like  

```
interface DisplayWitdth  {
  default: number;
  tablet?: number;
  desktop?: number;  
}
```
 also that will allow for adding something in the future like 
 
 ```
interface DisplayWitdth  {
  ... 
  bigPhone? : number;
  mediumPhone?: number;
  ... 
}
```